src/Dialogs/Label.vala

```diff
@@ -218,3 +218,4 @@ public class Dialogs.Label : Adw.Window {
         });
     }
 }
+
```

Comment:
```suggestion
```

Reply:
According to [Vala Lint](https://github.com/vala-lang/vala-lint/wiki/Vala-Lint-Checks) there should be a blank line at the end. Although it is not a problem, we still need to improve the code for that Lint to pass.